openapi_type/custom_types.py

```diff
@@ -68,19 +53,23 @@ def deserialize(self, node, cstruct: str) -> ContentTypeTag:
     def serialize(self, node, appstruct: ContentTypeTag) -> str:
         """ Converts ``ContentTypeTag`` back to string value suitable for JSON/YAML
         """
-        rv = [appstruct.format.value]
+        rv: List = [appstruct.format]
```

Comment:
does MyPy need a signature here? Seems like an obvious case as there's a list constructor

Reply:
Yes, it will report an error if we don't do this, `List` is equal to `List[Any]` here.
![image](https://user-images.githubusercontent.com/4213483/113012544-739d7200-91ad-11eb-8194-e91e82fcec98.png)